frontend/src/tests/components/EarthquakesTest/EarthquakesForm.test.js

```diff
@@ -0,0 +1,103 @@
+import { render, waitFor, fireEvent } from '@testing-library/react';
+import EarthquakesForm from 'main/components/Earthquakes/EarthquakesForm';
+import { earthquakesFixtures } from 'fixtures/earthquakesFixtures';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+const mockedNavigate = jest.fn();
+
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useNavigate: () => mockedNavigate,
+}));
+
+describe('EarthquakesForm tests', () => {
+  test('renders correctly ', async () => {
+    const { getByText } = render(
+      <Router>
+        <EarthquakesForm />
+      </Router>
+    );
+    // await waitFor(() => expect(getByText(/Quarter YYYYQ/)).toBeInTheDocument());
+    await waitFor(() => expect(getByText(/Retrieve/)).toBeInTheDocument());
+  });
+
+  // test('renders correctly when passing in an Earthquake', async () => {
+  //   const { getByText, getByTestId } = render(
+  //     <Router>
+  //       <EarthquakesForm
+  //         initialEarthquakes={earthquakesFixtures.twoEarthquakes}
```

Comment:
Maybe could have tested this by adding a fixture of a single earthquake? Not really sure if the functionality of ucsbdates and earthquakes are similar

Reply:
Not sure if this is needed since you're not creating any earthquake objects in form, just requesting earthquakes that match the distance

Reply:
Oh I see